sklearn/base.py

```diff
@@ -516,13 +546,30 @@ def score(self, X, y=None):
         pass
 
 
-###############################################################################
 class MetaEstimatorMixin(object):
+    _required_parameters = ["estimator"]
+
     """Mixin class for all meta estimators in scikit-learn."""
-    # this is just a tag for the moment
 
 
-###############################################################################
+class MultiOutputMixin(object):
+    """Mixin to mark estimators that support multioutput."""
+    def _get_tags(self):
+        return _update_tags(self, super(MultiOutputMixin, self),
+                            multioutput=True)
+
+
+def _is_32bit():
+    """Detect if process is 32bit Python."""
+    return struct.calcsize('P') * 8 == 32
+
+
+class _UnstableOn32BitMixin(object):
```

Comment:
Because it's a hack for numeric instabilities? I could make the other new mixins private, too?